assets/js/vue/components/search/SearchForm.vue

```diff
@@ -173,7 +173,7 @@
             route: function (value) {
                 if (!this.submitted) {
                     if (this.route) {
-                        const urlHashSplit = this.route.split("#")[1].split("&").map(value => value.split("="));
+                        const urlHashSplit = decodeURI(this.route).split("#")[1].split("&").map(value => value.split("="));
```

Comment:
Why const, and not `let`? Or `var`? is this a vue thing?

Reply:
Its just a regular Javascript constant, not a vue thing. If the value of variable isn't going to be reassigned we should use `const`. `const` and `let` are pretty much the same except that it can't be reassigned after it is declared